src/content/concepts/plugins.md

```diff
@@ -75,17 +76,22 @@ module.exports = config;
 
 ### Node API
 
-?> Even when using the Node API, users should pass plugins via the `plugins` property in the configuration. Using `compiler.apply` should not be the recommended way.
+When using the Node API, you can also pass plugins via the `plugins` property in the configuration.
 
 **some-node-script.js**
 
 ```javascript
   const webpack = require('webpack'); //to access webpack runtime
   const configuration = require('./webpack.config.js');
 
-  let compiler = webpack(configuration);
-  compiler.apply(new webpack.ProgressPlugin());
```

Comment:
You could use `new webpack.ProgressPlugin().apply(compiler)` instead

Reply:
Updated, thanks for the suggestion!